core-java-net/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is this an IDE-specific file, perhaps from Visual Studio? If so, then let's omit it from the repo.

Reply:
removed and rebase squash